src/components/modules/blockEvents.ts

```diff
@@ -52,6 +52,13 @@ export default class BlockEvents extends Module {
       case _.keyCodes.TAB:
         this.tabPressed(event);
         break;
+      case _.keyCodes.SLASH:
+        if (event.ctrlKey || event.metaKey) {
```

Comment:
Do we need both ctrl and meta keys?
On windows meta key is Win button

Reply:
yes, it's a standard statement to Cmd/Ctrl key